app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+        except VaccineError as e:
+            return str(e)
```

Comment:
and here you can finally write "all friends" because this function works for a list of friends